app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = " "
```

Comment:
nếu dùng " " như lúc e tạo thì lúc check .empty? sẽ là false nha em
```suggestion
  def full_title page_title = ""
```